tests/int/test_analyze.py

```
@@ -98,6 +106,58 @@ def test_taxonomy_search_analyzer(es_connection, data_ingester):
     assert _tokens(result) == ["en:organic"]
 
 
+@pytest.mark.xfail(reason="Because of the way we create indexes it does not work yet")
+def test_taxonomy_search_analyzer_without_synonyms(
+    es_connection, index_config, data_ingester
+):
+    def get_taxonomy_search_analyzer_no_synonyms(
+        config, taxonomy: str, lang: str, with_synonyms: bool
+    ):
+        import pdb
```

Ruff (0.16.0)

[error] Import for `pdb` found

(T100)